### PR TITLE
added discord markdown for description

### PR DESCRIPTION
**Pull request by thedarkrai07**
Fix for the Issue #213

### CODE REVIEW

File: packages/react-app/src/components/pages/Bounties/Bounty/index.tsx [hunk 1]
```diff
@@ -136,7 +139,7 @@ const BountyDetails = ({ bounty }: { bounty: BountyCollection }): JSX.Element =>
 			</GridItem>
 			<GridItem>
 				<Heading size="sm">Description</Heading>
-				<Text>{description}</Text>
+				<Text>{ ReactHtmlParser(DOMPurify.sanitize(toHTML(description))) }</Text>
 			</GridItem>
 			<GridItem>
 				<Heading size="sm">Done Criteria</Heading>
```
Comment (tescher) [inline packages/react-app/src/components/pages/Bounties/Bounty/index.tsx:145]: Can you add it to the next line also?

**Review by tescher — CHANGES_REQUESTED** [review]
Just one small addition

File: packages/react-app/src/components/pages/Bounties/Bounty/index.tsx (hunk 1 above)
Comment (thedarkrai07) [reply packages/react-app/src/components/pages/Bounties/Bounty/index.tsx:145]: You mean to indent it like this:
```				
<Text>
    { ReactHtmlParser(DOMPurify.sanitize(toHTML(description))) }
</Text> 
```

Comment (thedarkrai07) [reply packages/react-app/src/components/pages/Bounties/Bounty/index.tsx:145]: Do you mean I should add it to Done criteria?

Comment (tescher) [reply packages/react-app/src/components/pages/Bounties/Bounty/index.tsx:145]: Yes, sorry. Github doesn't allow me to put comments on lines below this window.

File: packages/react-app/src/components/pages/Bounties/Bounty/index.tsx [hunk 2]
```diff
@@ -136,11 +139,11 @@ const BountyDetails = ({ bounty }: { bounty: BountyCollection }): JSX.Element =>
 			</GridItem>
 			<GridItem>
 				<Heading size="sm">Description</Heading>
-				<Text>{description}</Text>
+				<Text>{ ReactHtmlParser(DOMPurify.sanitize(toHTML(description))) }</Text>
```
Comment (tescher) [inline packages/react-app/src/components/pages/Bounties/Bounty/index.tsx:142]: Need to check for description being null or undefined. Some bounties don't have descriptions or criteria, so these will be blank. Crashing:
image

**Review by tescher — CHANGES_REQUESTED** [review]
One more thing after testing

**Review by Behold3th — APPROVED** [review]

**Pull request merged**